### DISCUSSION TITLE
Add key stats on telegram notification

### DISCUSSION THREAD

**Discussion by MeninSun** [masterking32/MasterHamsterKombatBot#230] [Ideas]
Hi, currently i using this config :
-----------------------------------------------
![image](https://github.com/user-attachments/assets/c1191312-2a4c-49ce-8a56-90eef6bc01be)

Can you add key stats on telegram notification? For example will display like this : 

If all key not completed :
[Account 1] Apply new key / whatever it will show

If all key completed :
[Account 1] All key done

And the final information :
[Account 1] Final account balance: 47.88m coins, Your profit per hour is 2.61m ( 0), Spent: 0, Total key : xxx

But the important, i just need key stats on final information that will send to telegram, not on terminal only.

Thanks you, and good job for your work 👍 💯 🥇

**Comment by Homavi88** [comment]
I am also interested in the life of a bot via telegram
